portfolio/src/main/webapp/script.js

```diff
@@ -137,15 +137,49 @@ function getMessage() {
 function handleResponse(response) {
   const textPromise = response.text();
   
-  textPromise.then(addMessageToDom);
+  textPromise.then(addSingleMessageToDom);
 }
 
 /**
- * Adds the given message to the DOM.
- * @param {string} message The text to be added inside of the div 
+ * Adds a single message {@code message} inside of the div
        message-container. 
```

Comment:
I believe you need to prepend this text with *

Reply:
Thanks for catching that. I made the changes.

Done!